virtcontainers/qemu_arm64.go

```diff
@@ -20,7 +23,7 @@ const defaultQemuPath = "/usr/bin/qemu-system-aarch64"
 
 const defaultQemuMachineType = QemuVirt
 
-const defaultQemuMachineOptions = "gic-version=host,usb=off,accel=kvm"
+var defaultQemuMachineOptions = "usb=off,accel=kvm" + "," + guestGICversion
```

Comment:
Can it just be "usb=off,accel=kvm,"

Reply:
@caoruidong Yes, that would be better, I will fix it.